browser/socket.js

```diff
@@ -56,24 +55,35 @@ socket.on('usersUpdated', users => {
     // These values are passed up as "scene" in the user tick and correspond to the names of react components and a-scenes
     const currentScene = window.location.pathname.replace(/\//g, '') || 'root';
     // If the user is on the current scene, add or update the user's head and body
-    const avatarHead = document.getElementById(user.get('id'));
-    const avatarBody = document.getElementById(`${user.get('id')}-body`);
     if (user.get('scene') === currentScene) {
-      // console.log(`Attempting to update ${user.get('id')}-body`);
+      const avatarHead = document.getElementById(user.get('id'));
+      const avatarBody = document.getElementById(`${user.get('id')}-body`);
```

Comment:
Any particular reason you've block-scoped `avatarHead` and `avatarBody`? (Not criticizing, just genuinely curious...instead of searching through the DOM twice, this section has to do it 4 times now. Just seems a tad less performant to me.)